info/contrib.md

```diff
@@ -28,15 +28,17 @@ Finally, feel free to hop on [#rust-gamedev](http://chat.mibbit.com/?server=irc.
 
 ### Directory Structure 
 
-* _examples_ : GFX's examples
-* _tests_ : GFX's tests
-* _info_ : Information and documentation
-* _src_ : gfx_app, an application framework for GFX
-* _src/core_ : gfx_core, core structures and the interface that backends must provide
-* _src/backend_ : The backends implementations
-* _src/render_ : The main gfx package
+* _src/backend : The different backends GFX supports.
+* _src/core : gfx_core, core structures and the interface that backends must provide
```

Comment:
We need to be consistent here by either naming the crates explicitly for all entries, or omitting them for all (simpler).
Also, trailing point is missing at the end.